variables.tf

```diff
@@ -214,14 +214,18 @@ variable "auto_scaling" {
 
 variable "kms_key_crn" {
   type        = string
-  description = "(Optional) The root key CRN of a Key Management Service like Key Protect or Hyper Protect Crypto Service (HPCS) that you want to use for disk encryption. If `null`, database is encrypted by using randomly generated keys. See https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-key-protect&interface=ui#key-byok for current list of supported regions for BYOK"
+  description = "(Optional) The root key CRN of a Key Management Service like Key Protect or Hyper Protect Crypto Service (HPCS) to use for disk encryption. If `null`, database is encrypted by using randomly generated keys. Refer to the [disk encryption](https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-key-protect#using-the-key-protect-key) documentation for more information"
   default     = null
 }
 
 variable "backup_encryption_key_crn" {
   type        = string
-  description = "(Optional) The CRN of a Key Protect Key to use for encrypting backups. If left null, the value passed for the 'kms_key_crn' variable will be used. Take note that Hyper Protect Crypto Services for IBM Cloud® Databases backups is not currently supported."
+  description = "(Optional) The CRN of a key protect key to use for encrypting the disk that holds deployment backups. BYOK for backups is available with keys from us-south, us-east or eu-de. Important: Only keys in us-south and eu-de are durable to region failures, refer to the [BYOK](https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-key-protect&interface=ui#key-byok) documentation for more information. If null, will use 'key_protect_key_crn' as encryption key. If 'key_protect_key_crn' is also null database is encrypted by using randomly generated keys."
```

Comment:
This description needs to support Hyper protect

Reply:
AFAIK, hyper protect is not supported for backup encryption. Am I missing something?

Under Key Protect (https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-key-protect&interface=ui#key-byok) it says "If you use Key Protect" in the backup section.

There is no mention of backups on the Hyper Protect documentation (https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-hpcs&interface=ui). The implication being it is not support.

Further the ICD backups documentation (https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-dashboard-backups&interface=ui) says `Backup storage is encrypted. To manage the encryption keys, see Key Protect integration. Otherwise, backups are encrypted with a key that is automatically generated for your deployment.`. Again, the implication is either Key Protect or internal to ICD, so no support for Hyper Protect.

Reply:
This is resolved for now. An issue has been raised against ICD to support hs-crypto at which point this will need revisiting and updating.